src/SpaceChat.ts

```diff
@@ -0,0 +1,171 @@
+import axios from 'axios'
+import fs from 'fs'
+import path from 'path'
+import readline from 'readline'
+import winston from 'winston'
+import { WebSocket } from 'ws'
+import { Downloader } from './Downloader'
+import { SpaceChatKind } from './enums/SpaceChatKind.enum'
+import { logger as baseLogger } from './logger'
+
+export class SpaceChat {
+  private readonly WS_URL = 'wss://prod-chatman-ancillary-ap-northeast-1.pscp.tv/chatapi/v1/chatnow'
+  private readonly CHAT_API_URL = 'https://proxsee.pscp.tv/api/v2/accessChatPublic'
+
+  private logger: winston.Logger
+  private ws: WebSocket
+  private accessChatData: Record<string, any>
+
+  private tmpChatFile: string
+  private outChatFile: string
+
+  constructor(
+    public spaceId: string,
+    private liveStreamStatus: Record<string, any>,
+    private options: Record<string, any>,
+  ) {
+    this.logger = baseLogger.child({ label: `[SpaceChat@${spaceId}]` })
+    this.spaceId = spaceId
+    this.liveStreamStatus = liveStreamStatus
+    this.options = options
```

Comment:
Setting `private` in the constructor already makes them fields. No need to do this again.